Documentation/migration-guide.md

```diff
@@ -148,6 +148,23 @@ MobileCore.registerExtensions([
 
 ------
 
+### Granular Ad Tracking
+
+```diff
+- let tracker = Media.createTracker()
++ var trackerConfig: [String: Any] = [:]
++ trackerConfig[MediaConstants.TrackerConfig.AD_PING_INTERVAL] = 1
++ let tracker = Media.createTrackerWith(config: trackerConfig)
+
+guard var mediaObject = guard let mediaObject = Media.createMediaObjectWith(name: "name", id: "id", length: 30, streamType: "vod", mediaType: MediaType.Video) else {
+  return
+}
+- mediaObject[MediaConstants.MediaObjectKey.GRANULAR_AD_TRACKING] = true
```

Comment:
can you add a quick description under the title about what was this setting doing in Media extension and add a hint about the interval supported? e.g.
With AEPMedia extension you were able to set granular ad content tracking to 1 second by enabling the `GRANULAR_AD_TRACKING` setting. With AEPEdgeMedia you need to configure the ad content tracking interval per tracker, setting `AD_PING_INTERVAL` to a value between 1 and 10 seconds.